src/components/ha-service-description.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+
+import { HomeAssistant } from "../types";
+
+@customElement("ha-service-description")
+class HaServiceDescription extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property() public domain!: string;
+
+  @property() public service!: string;
+
+  protected render(): TemplateResult {
+    return html` ${this._getDescription()} `;
```

Comment:
We might as well make this a helper returning a string instead of an element?

Reply:
Had that in the back of my head asking "why is this an element"? Should have listened 😜 agreed, I'll make the change

Reply:
I might back this one off. I don't really have a good way to test zwave integration UI changes if I'm replacing the actual usage. Maybe I could mock the integration setup? Not sure?